crates/vault-registry/src/types.rs

```diff
@@ -814,6 +831,10 @@ impl<T: Config> UpdatableVault<T> for RichSystemVault<T> {
         )
     }
 
+    fn control(&self) -> T::AccountId {
+        Pallet::<T>::liquidation_vault_account_id()
```

Comment:
This doesn't really make sense for the liquidation vault. Would it to move this function to the `RichVault` impl, rather than the trait impl?